staff/judith-gongora/cv-style/index.html

```diff
@@ -0,0 +1,110 @@
+                    <div class="name"> <h1>Judith Góngora</h1><p>Full Stack developer in the future</p></div>
+                    <div>
+                    <address>
+                        Website: 
+                        <a href="http://www.google.com" title="Personal website">http://www.google.com</a>
+                        <br>
+                        Email: 
+                        <a href="mailto:google@gmail.com" title="Email">google@gmail.com</a>
+                        <br>
+                        Phone: 
+                        +34 666 66 66 66
+                        <br>
+                        Location: 
+                        Barcelona
+                    <address>
+                    </div>
+                </section>
+                <hr>
+            </div>
+            <div class="main__profile">
+                <section class="profile">
+                    <div class="title"><h2>Profile Summary</h2></div>
+                    <div class="p-profile"> <p>IT with 8 years of experience that seeks to expand horizons professionally in the world of web development</p>
+                    </div>
+                </section>
+                <hr>
+            </div>
+            <div class="main__interest">
+                <section class="interest">
+                    <div class="title"><h2>Interest</h2></div> 
+                    <div class="flex-interest">
+                        <div class="-group"><h4>Objetive</h4>
+                        <p>Competence development in the field of information technology. Expansion of professional experience.</p>
+                        </div>
+                        <div class="-group1"> <h4>Languajes</h4>
+                        <p>Catalan - native <br>Spanish - native <br>English - elementary</p>
```

Comment:
having different paragraphs here may be a better option than breaking one with br tags